mscore/plugin/api/part.h

```diff
@@ -27,6 +27,8 @@ class Part : public Ms::PluginAPI::ScoreElement {
       Q_OBJECT
       Q_PROPERTY(int                            startTrack          READ startTrack)
       Q_PROPERTY(int                            endTrack            READ endTrack)
+      /// The string identifier for the current instrument.
+      Q_PROPERTY(QString                        instrumentId        READ instrumentId) /// \since MuseScore 3.2
```

Comment:
I suspect Doxygen won't recognize this correctly. It would be better to put this `\since` statement right to the documentation comment a line above, something like:
`/// The string identifier for the current instrument. \since MuseScore 3.2`

Reply:
Done.